test/test_integration_credentials_netrc.py

```diff
@@ -325,46 +326,17 @@ def test_https_netrc_does_not_remove_irrelevant_entry(self,mocks):
 
         self.invoke_cli(self.cli_auth_params,
                         ['logout'])
-        cred_store, _ = CredentialStoreFactory().create_credential_store()
+        cred_store = utils.create_cred_store()
         assert cred_store.is_exists("somemachine")
         assert not cred_store.is_exists(self.client_params.hostname)
 
-    @integration_test(True)
-    def test_https_logout_twice_returns_could_not_logout_message(self,mocks):
-        utils.setup_cli(self)
-        self.invoke_cli(self.cli_auth_params,
-                        ['login', '-i', 'admin', '-p', self.client_params.env_api_key])
-
-        self.invoke_cli(self.cli_auth_params,
-                        ['logout'])
-
-        unsuccessful_logout = self.invoke_cli(self.cli_auth_params,
-                                              ['logout'], exit_code=1)
-
-        self.assertIn("Failed to log out. You are already logged out", unsuccessful_logout.strip())
-
-    '''
-        Validates when a user can logout successfully
     '''
-
-    @integration_test(True)
-    def test_https_logout_successful(self,mocks):
-        utils.setup_cli(self)
-        self.invoke_cli(self.cli_auth_params,
-                        ['login', '-i', 'admin', '-p', self.client_params.env_api_key])
-
-        output = self.invoke_cli(self.cli_auth_params,
-                                 ['logout'])
-
-        self.assertIn('Successfully logged out from Conjur', output.strip())
-
-    '''
-        Validates that if a user configures the CLI in insecure mode and runs the command not in 
-        insecure mode, then they will fail
+    Validates that if a user configures the CLI in insecure mode and runs the command not in 
+    insecure mode, then they will fail
     '''
 
     @integration_test(True)
-    def test_cli_configured_in_insecure_mode_but_run_in_secure_mode_raises_error(self,mocks):
+    def test_cli_configured_in_insecure_mode_but_run_in_secure_mode_raises_error_netrc(self, mocks):
```

Comment:
Can you add the updated test names to this spreadsheet: https://cyberark365-my.sharepoint.com/:x:/r/personal/elad_kugman_cyberark_com/_layouts/15/Doc.aspx?sourcedoc=%7B3F985515-C4C3-477A-9E98-6BEFBBDEC660%7D&file=dockerless_testplan.xlsx&action=default&mobileredirect=true&cid=6eb95c20-7b32-451c-8fd0-2db6c62fb841

That way me and @eladkug and you can ensure we are aligned